### PR TITLE
Edit README (minor) and use Angular code style in examples

### PR DESCRIPTION
**Pull request by oboukli**
Summary:

- Fix Markdown lint issues
- NPM no longer has `--save` option
- Minor edits

### CODE REVIEW

File: docs/store/README.md [hunk 1]
```diff
@@ -36,19 +36,21 @@ combination of these reducers will make up your application state:
 // counter.ts
 import { Action } from '@ngrx/store';
 
-export const INCREMENT = 'INCREMENT';
-export const DECREMENT = 'DECREMENT';
-export const RESET = 'RESET';
+export const increment = 'INCREMENT';
```
Comment (tdeschryver) [inline docs/store/README.md]: I think the all caps variable names are desired here.

File: docs/store/README.md [hunk 2]
```diff
@@ -67,7 +69,7 @@ import { StoreModule } from '@ngrx/store';
 import { counterReducer } from './counter';
 
 @NgModule({
-  imports: [BrowserModule, StoreModule.forRoot({ count: counterReducer })],
+  imports: [StoreModule.forRoot({ count: counterReducer })],
```
Comment (tdeschryver) [inline docs/store/README.md]: Why did you remove `BrowserModule`?

Comment (oboukli) [reply docs/store/README.md]: It is not imported, and it is not required to understand the example.

We can include the full `AppModule` which complements the rest of the code snippets in this document, but I prefer to keep it simple.

Here's how the full `AppModule` looks like:

```ts
import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';
import { StoreModule } from '@ngrx/store';
import { AppComponent } from './app.component';
import { counterReducer } from './counter';
import { MyCounterComponent } from './my-counter.component';

@NgModule({
  declarations: [
    AppComponent, MyCounterComponent
  ],
  imports: [
    BrowserModule,
    StoreModule.forRoot({ count: counterReducer })
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```

File: docs/store/README.md (hunk 1 above)
Comment (oboukli) [reply docs/store/README.md]: This is an archaic style that shouldn't infect ngrx.
https://angular.io/guide/styleguide#style-03-02

Comment (brandonroberts) [reply docs/store/README.md]: Keep the constants in all caps

**Comment by brandonroberts** [comment]
Thanks!
